kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/commands/CSVSchemaCommand.java

```diff
@@ -79,6 +79,9 @@ public CSVSchemaCommand(Logger console) {
   @Parameter(names="--skip-lines", description="Lines to skip before CSV start")
   int linesToSkip = 0;
 
+  @Parameter(names={"-n", "--samples"}, description="Number of lines to sample when inferencing schema")
```

Comment:
We just added --num-records to the JSON equivalent. Could you copy that so the doc and options match?